src/doc/manifest.md

```diff
@@ -54,6 +56,15 @@ the VCS's ignore settings (`.gitignore` for git for example).
 exclude = ["build/**/*.o", "doc/**/*.html"]
 ```
 
+```toml
+[package
+# ...
+include = ["src/resource.data"]
```

Comment:
I think that this means that `cargo package` will *only* include `src/resource.data`, which is to say I think that `include` must be an exhaustive list of files.